assets/themes/default/fonts/Vazir_WOL/Vazir_WOL.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+    font-family: Vazir;
+    src: url('Vazir-WOL.eot');
+    src: url('Vazir-WOL.eot?#iefix') format('embedded-opentype'),
+        url('Vazir-WOL.woff2') format('woff2'),
+        url('Vazir-WOL.woff') format('woff'),
+        url('Vazir-WOL.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
```

Comment:
we could add `unicode-range` here
unfortunately, those being the same as Arabic, might break Arabic texts.
@DearRude, what do you think?

Reply:
Just like English and German, Persian and Arabic have common characters. We would not be able to cut down Arabic by `unicode-range`.
However, I suggest using Vazir and Shamim fonts for Arabic too. Since those fonts fully support Arabic and have been used in other projects in Arabic, I won't be a bad idea if we support both.

I can provide range of unicode in Persian and Arabic together if needed.

Reply:
the ranges are well documented, https://en.m.wikipedia.org/wiki/Arabic_script_in_Unicode but excessive
so i think we should use the non-Latin variants of the scripts.

Reply:
This ```Vazir-WOL``` is a non-latin package. But as @dearrude said, Persian and Arabic has some common characters.

Reply:
WOL ➡️ Without Latin

so how come @elegaanz noticed those issues with the centering of the text? what's going on?

Reply:
As @elegaanz said, It's an strange issue and I don't know if ```Shabnam``` could fix it or not!

Reply:
let's try!

Reply:
```Shabnam``` Added. @igalic @elegaanz